### DISCUSSION TITLE
Idea about testing environment

### DISCUSSION THREAD

**Discussion by mesfandi** [elsa-workflows/elsa-core#1380] [Ideas]
Hi,
Thank you for your nice work. Just as an idea for future features, it would be awesome if you could add a Test (Run) environment on the UI, where users can specify a dummy input and test their workflows as they are creating them.
Thank you so much.

**Comment by sfmskywalker** [comment]
Thanks for the suggestion! Yes, we definitely want this feature. In fact, we have started the design phase on it and some initial development has started. It will be an incremental process where we start with the basics and improve & add things over time.
